src/GameFace/Struct/UserProfile.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace GameFace.Struct
+{
+    public static class Helper
+    {
+        public static List<UserRecords> AddElement(List<UserRecords> list, UserRecords element)
+        {
+            var found = false;
+            if (list.Count == 0)
+            {
+                list.Add(element);
+            }
+            else
+            {
+                for (var i= 0; i < list.Count; i++)
+                {
+                    if (!list[i].Task.Equals(element.Task)) continue;
+                    var newrecord = new UserRecords(list[i].Task, list[i].NoPerforms + 1, list[i].Value + element.Value);
+                    list.Remove(list[i]);
+                    list.Add(newrecord);
+                    found = true;
+                }
+                if (!found)
+                {
+                    list.Add(element);
+                }
+            }
+            
+            return list;
+        }
+
+
+        public static List<SprintRecords> DevideBySprints(List<UserRecordsDate> list)
+        {
+            var date = list[0].Date;
```

Comment:
Correct me if I am wrong, but this method assumes that the list of UserRecords is received sorted in descending order. While for the case you are using it, those conditions will always be true, it might not be the case for future usages.

Let's say that tomorrow I write some code, and I want to use your DevideBySprints method. I need to have the knowledge that it needs the list in descending order, which will probably not be the case.

I recommend that we do the OrderByDescending here - that way we idiot proof the method for future developers that want to use it.